src/pages/Dashboard/UpcomingRuns-Tile.vue

```diff
@@ -407,60 +444,52 @@ export default {
             </v-list-item-avatar>
           </v-list-item>
         </v-lazy>
+      </v-list>
+    </v-card-text>
 
-        <v-btn
-          text
-          color="deepRed"
-          small
-          class="position-absolute"
-          :style="{ bottom: '8px', right: '4px' }"
-          tile
-          :loading="isClearingLateRuns"
-          @click="handleOpenDialog"
-        >
-          Clear
-        </v-btn>
+    <v-spacer />
 
-        <v-dialog v-model="showClearLateRunsDialog" max-width="480">
-          <v-card flat>
-            <v-card-title class="text-h6 word-break-normal">
-              Are you sure you want to clear all late runs?
-            </v-card-title>
-
-            <v-card-text v-if="scheduledRunIds.length > 1">
-              This will toggle the schedule for all flows with late runs. If you
-              did not set a
-              <ExternalLink
-                href="https://docs.prefect.io/core/concepts/schedules.html#schedules"
-                >start time</ExternalLink
-              >
-              for your schedule,
-              <strong
-                >it may affect the scheduled start times for your upcoming
-                runs.</strong
-              >
-            </v-card-text>
-            <v-card-text v-else>
-              This will set your late flow runs into a
-              <strong>cancelled</strong> state.
-            </v-card-text>
-
-            <v-card-actions>
-              <v-spacer></v-spacer>
-              <v-btn text tile @click="showClearLateRunsDialog = false">
-                Cancel
-              </v-btn>
-              <v-btn dark color="deepRed" depressed @click="clearLateRuns">
-                Confirm
-              </v-btn>
-            </v-card-actions>
-          </v-card>
-        </v-dialog>
-      </v-list>
+    <v-card-actions class="py-0">
+      <v-spacer />
 
-      <div v-if="lateRuns && lateRuns.length > 3" class="pa-0 card-footer">
-      </div>
-    </v-card-text>
+      <v-btn
+        v-if="!overlay && lateRuns && lateRuns.length > 0"
+        small
+        depressed
+        color="primary"
+        text
+        style="z-index: 2;"
+        @click="showOverlay('late')"
+      >
+        Clear late
+      </v-btn>
+
+      <v-btn
+        v-if="!overlay"
+        small
+        depressed
+        color="primary"
+        text
+        style="z-index: 2;"
+        @click="showOverlay('queue')"
+      >
+        Options
```

Comment:
Hm, that wording is intentional for now to avoid the feeling that the "Pause work queue" would happen with just 1 click; in this case it indicates that there's more to do. 

Wrt slowness: I think that's a fairly common use case so let me investigate what's going on there, I'd want the overlay to show regardless of the number of runs.

Reply:
Ok found the issue with the slowness, should be much improved now: basically when adding the overlay I used a `v-if`, which completely removes anything under it when the condition isn't met. Using `v-show` prevents us from having to re-render the entire list everytime the overlay is pressed. 👍🏻